addons/website_slides/security/website_slides_security.xml

```diff
@@ -90,15 +117,62 @@
+                        '&amp;',
+                            ('channel_id.visibility', '=', 'public'),
+                            '|',
+                            '|',
+                                ('channel_id.partner_ids', 'in', user.partner_id.ids),
+                                ('is_category', '=', True),
+                                ('is_preview', '=', True),
+                        ('channel_id.partner_ids', 'in', user.partner_id.ids),
+                        '&amp;',
+                            ('channel_id.visibility', '=', 'logged_user'),
+                            '|',
+                            '|',
+                                ('channel_id.partner_ids', 'in', user.partner_id.ids),
+                                ('is_category', '=', True),
+                                ('is_preview', '=', True),
+                    '&amp;',
+                        ('channel_id.website_published', '=', True),
+                        '|',
+                            ('user_id', '=', user.id),
+                            ('website_published', '=', True)
+                ]
+            </field>
+            <field name="perm_unlink" eval="0"/>
+            <field name="perm_write" eval="0"/>
+            <field name="perm_read" eval="1"/>
+            <field name="perm_create" eval="0"/>
+        </record>
+
         <record id="rule_slide_slide_not_website" model="ir.rule">
             <field name="name">Slide: public/portal/user: restricted to published or uploaded by user, and either channel member or public channel &amp; (category or previewable)</field>
             <field name="model_id" ref="model_slide_slide"/>
-            <field name="groups" eval="[(4, ref('base.group_public')), (4, ref('base.group_portal')), (4, ref('base.group_user'))]"/>
-            <field name="domain_force">['&amp;',
-    '|',
-        '&amp;', ('channel_id.visibility', '=', 'public'), '|', ('is_category','=', True), ('is_preview', '=', True),
-        ('channel_id.partner_ids', '=', user.partner_id.id),
-    '&amp;', ('channel_id.website_published', '=', True), '|', ('user_id', '=', user.id), ('website_published', '=', True)]</field>
+            <field name="groups" eval="[(4, ref('base.group_public'))]"/>
+            <field name="domain_force">[
+                '&amp;',
+                    '&amp;',
+                        ('channel_id.website_published', '=', True),
+                        '|',
+                            ('user_id', '=', user.id),
+                            ('website_published', '=', True),
+                    '|',
+                        ('channel_id.partner_ids', '=', user.partner_id.id),
+                        '&amp;',
+                            ('channel_id.visibility', '=', 'public'),
+                            '|',
+                                ('is_category','=', True),
+                                ('is_preview', '=', True),
+                ]
+            </field>
```

Comment:
Also here, if the rule only applies to public users now, can we stick to public & published & preview/category?

```suggestion
            <field name="domain_force">[
                '&amp;',
                    '&amp;',
                        ('channel_id.website_published', '=', True),
                        ('website_published', '=', True),
                    '&amp;',
                        ('channel_id.visibility', '=', 'public'),
                        '|',
                            ('is_category','=', True),
                            ('is_preview', '=', True),
                ]
            </field>
```